### ISSUE TITLE
Does this library allow you to post to instagram

### ISSUE THREAD

**Issue by eQX8HNMTU2** [cosenary/Instagram-PHP-API#250]
Is there some way I can post an image with this library to instagram?

**Comment by johannesnagl** [comment]
No, therefore, you need to be a Facebook Marketing Partner(*) and use the new Instagram Graph API version. You can't do this with the legacy Instagram API.

(*) At the time of writing this comment.
